check50/internal.py

```diff
@@ -35,6 +35,13 @@ class Error(Exception):
     pass
 
 
+class RemoteCheckError(Error):
```

Comment:
Should this be in `internal`? internal is traditionally the place for the extension-writer API. `RemoteCheckError` isn't part of that API.

Reply:
I agree, but I don't think we have a better place to put it right now. Perhaps we should introduce `exceptions.py`? And move `Error`, `RemoteCheckError` and `_excepthook` to that instead?

Reply:
inb4 `truly_internal_dont_touch.py`